HarfangLab/action_harfanglab_gethostnamesbyip.json

```
@@ -0,0 +1,36 @@
+{
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+      "target_ip": {
+        "description": "Targeted IP address",
+        "type": "string"
+      },
+      "get_only_last_seen": {
+        "description": "Get the last seen hostname only",
+        "type": "boolean"
+      }
+    },
+    "required": [],
+    "title": "Arguments",
+    "type": "object"
+  },
+  "description": "Get the hostname of a machine by its IP address",
+  "docker_parameters": "harfanglab_get_hostnames_by_ip",
+  "name": "Hostname by IP",
+  "results": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+      "hostnames": {
+        "description": "Hostnames",
+        "type": "object"
```

According to the code, `hostnames` is a list of object

```suggestion
        "description": "Hostnames",
        "type": "array",
        "items": {
            "type": "object"
        }
```